### PR TITLE
Updating contributing.md file and Added contributing guidelines 

### LINKED ISSUE #42: Updating contributing guidelines
### **Feature request**:

Updating contributing guidelines

### **Is your feature request related to a problem? Please describe. (optional)**:

I want to add more information in the contributing.md file

### **Describe the solution you'd like**:

Adding for a perfect workflow :)

### **Describe how this feature will be useful to our readers**:

They will understand the guidelines more accurately.

<hr/>

**Note:**

- If you want to work on an issue, you should check if it has already been assigned to anyone.
  Yes, I have checked

  
  These are the things I want to add to the file.
  Kindly assign me issue :)

image

### PR DESCRIPTION
**Pull request by ayushsoni1010**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
- added guidelines to the contributing.md file

Fixed #42 

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] I have pulled the latest changes in the repository
- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by shriya030 — CHANGES_REQUESTED** [review]
We don't have the PR template which you have put in this. Kindly change that.

File: contributing.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ Please take a moment to review this document in order to make the contribution p
...
 Happy Contributing :slightly_smiling_face:
 
+# Contributing Guidelines  
+  
+This documentation contains a set of guidelines to help you during the contribution process.
+ 
+We are happy to welcome all the contributions from anyone willing to improve this project. Thank you for helping out and remember, **no contribution is small.** 
+ 
+- Participants are expected to follow project guidelines and coding style.
+- Each participant/contributor will be assigned **2 issues (max)** at a time to work on.
```
Comment (shriya030) [inline contributing.md]: Remove line no. 16

Comment (ayushsoni1010) [reply contributing.md]: okay, wait a min

Comment (ayushsoni1010) [reply contributing.md]: Done

**Comment by ayushsoni1010** [comment]
Alright ?

**Review by shriya030 — APPROVED** [review]

**Review by aj3011 — APPROVED** [review]

**Review by shriya030 — APPROVED** [review]

**Pull request merged**